api/src/index.js

```diff
@@ -11,21 +11,81 @@ app.use(bodyParser.json());
 
 const apiRouter = express.Router();
 
-// You can delete this route once you add your own routes
-apiRouter.get("/", async (req, res) => {
-  const SHOW_TABLES_QUERY =
-    process.env.DB_CLIENT === "pg"
-      ? "SELECT * FROM pg_catalog.pg_tables;"
-      : "SHOW TABLES;";
-  const tables = await knex.raw(SHOW_TABLES_QUERY);
-  res.json({ tables });
+// Respond with all meals in the future (relative to the when datetime)
+app.get("/future-meals", async (req, res, next) => {
+  try {
+    const now = new Date().toISOString();
+    const futureMeals = await knex("meal").where("when", ">", now);
+    res.json(futureMeals);
+  } catch (error) {
+    next(error);
+  }
+});
+
+// Respond with all meals in the past (relative to the when datetime)
+app.get("/past-meals", async (req, res, next) => {
+  try {
+    const now = new Date().toISOString();
+    const pastMeals = await knex("meal").where("when", "<", now);
```

Comment:
A suggestion would be to use knex `now` helper method to avoid being explicit about the date format

https://knexjs.org/guide/utility.html#now

```suggestion
    const pastMeals = await knex("meal").where("when", "<", knex.fn.now());
```